docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -105,105 +112,95 @@ significant changes in each minor release.
 ============================================= ======================================================
 devel                                         In development (ansible-core 2.11 unreleased, trunk)
 `2.10 ansible-base Release Notes`_            Maintained (security **and** general bug fixes)
+`2.9 Release Notes`_                          Maintained (pre-collections) (security **and** critical bug fixes)
+`2.8 Release Notes`_                          Maintained (pre-collections) (security fixes only)
+`2.7 Release Notes`_                          Unmaintained (end of life)
+`2.6 Release Notes`_                          Unmaintained (end of life)
+`2.5 Release Notes`_                          Unmaintained (end of life)
+<2.5                                          Unmaintained (end of life)
 ============================================= ======================================================
 
-
 .. _2.10 ansible-base Release Notes:
 .. _2.10-base: https://github.com/ansible/ansible/blob/stable-2.10/changelogs/CHANGELOG-v2.10.rst
+.. _2.9 Release Notes:
+.. _2.9: https://github.com/ansible/ansible/blob/stable-2.9/changelogs/CHANGELOG-v2.9.rst
+.. _2.8 Release Notes:
+.. _2.8: https://github.com/ansible/ansible/blob/stable-2.8/changelogs/CHANGELOG-v2.8.rst
+.. _2.7 Release Notes: https://github.com/ansible/ansible/blob/stable-2.7/changelogs/CHANGELOG-v2.7.rst
+.. _2.6 Release Notes:
+.. _2.6: https://github.com/ansible/ansible/blob/stable-2.6/changelogs/CHANGELOG-v2.6.rst
+.. _2.5 Release Notes: https://github.com/ansible/ansible/blob/stable-2.5/changelogs/CHANGELOG-v2.5.rst
 .. _support_life:
 .. _methods:
 
-.. _development_and_stable_version_maintenance_workflow:
+Preparing for a new release
+---------------------------
 
-Development and stable version maintenance workflow
------------------------------------------------------
+.. _release_freezing:
 
-The Ansible community develops and maintains Ansible and ``ansible-core`` on GitHub_.
+Feature freezes
+^^^^^^^^^^^^^^^
 
-Collection updates (new modules, plugins, features and bugfixes) will always be integrated in what will become the next version of Ansible. This work is tracked within the individual collection repositories.
+While there is a pending release candidate, core developers and maintainers focus on improving the release candidate, not on adding or reviewing new features.
```

Comment:
Ansible roadmap says feature freeze happens at beta. ansible-core roadmap does not specify a feature freeze time.

Reply:
I made this more general - "in the final preparations for a release" so the section is more of a general warning that we may not merge your PR immediately.